pkg/kubelet/types/labels.go

```diff
@@ -17,24 +17,32 @@ limitations under the License.
 package types
 
 const (
-	KubernetesPodNameLabel       = "io.kubernetes.pod.name"
-	KubernetesPodNamespaceLabel  = "io.kubernetes.pod.namespace"
-	KubernetesPodUIDLabel        = "io.kubernetes.pod.uid"
+	// KubernetesPodNameLabel specifies a pod name label
+	KubernetesPodNameLabel = "io.kubernetes.pod.name"
+	// KubernetesPodNamespaceLabel specifies a pod namespace label
+	KubernetesPodNamespaceLabel = "io.kubernetes.pod.namespace"
+	// KubernetesPodUIDLabel specifies a pod UID label
+	KubernetesPodUIDLabel = "io.kubernetes.pod.uid"
+	// KubernetesContainerNameLabel specifies a container name label
 	KubernetesContainerNameLabel = "io.kubernetes.container.name"
 )
 
+// GetContainerName return container name from labels
```

Comment:
Nit: For all the functions in this file, I think we want `FuncName returns ...`.

Reply:
fixed